packages/hoprd/rest-api-v2-spec.yaml

```diff
@@ -53,6 +53,16 @@ components:
         - version
       example:
         version: '1.83.5'
+    SignedMessage:
+      type: object
+      properties:
+        signedMessage:
+          type: string
+          description: Signature from requested message
```

Comment:
So its not a message but rather signature. Should the field not be called `signature` as well?

Reply:
Actually that's not a bad point, let me update it.

Reply:
Updated in 1f47b50